rust-runtime/aws-smithy-client/src/lib.rs

```diff
@@ -226,7 +228,47 @@ where
             .layer(DispatchLayer::new())
             .service(connector);
 
-        check_send_sync(svc).ready().await?.call(input).await
+        // send_operation records the full request-response lifecycle.
+        // NOTE: For operations that stream output, only the setup is captured in this span.
```

Comment:
Would users want to capture the streaming stuff in this span? If yes, is there anything they can do?

Reply:
There's no way to include the streaming body without consuming it, so no.